src/lib/components/tablev2/Search.js

```diff
@@ -0,0 +1,92 @@
+//@flow
+import React from 'react';
+import styled from 'styled-components';
+import { useTableContext } from './Tablev2.component';
+import SearchInput from '../searchinput/SearchInput.component';
+import { BasicText } from '../text/Text.component';
+import { spacing } from '../../style/theme.js';
+
+export type DisplayedName = {
+  plural: string,
+  singular: string,
+};
+
+export type SearchProps = {
+  onChange: (string) => void,
+  value?: string,
+  displayTotalOf?: boolean,
+  displayedName?: DisplayedName,
+  locale?: 'en' | 'fr',
+};
```

Comment:
It should extends the SearchInput props I guess. 
```suggestion
export type SearchProps = {
  onChange: (string) => void,
  displayTotalOf?: boolean,
  displayedName?: DisplayedName,
  locale?: 'en' | 'fr',
} & SearchInputProps;
```